project/REVIEWING.md

```diff
@@ -151,7 +151,7 @@ leave a comment to explain what documentation changes you think might be needed.
 - If the PR has the `impact/documentation` label, merging is delayed until a
   documentation maintainer acknowledges that a corresponding documentation PR
   (or issue) is opened on the documentation repository. Once a documentation
-  maintainer acknowledges the change, she/he will move the PR to `status/4-merge`
+  maintainer acknowledges the change, they/they will move the PR to `status/4-merge`
```

Comment:
Yup, this one we should change; the bot / algorithm could be improved here though, as just `they` should suffice here 😅